docs/gen-2/gold-silver/catext/nsc/README.md

```diff
@@ -0,0 +1,257 @@
+
+Fight Ralph if you have killed an encounter: Leer, Scratch 3x
+
+Teach Mud Slap and fight Daniel if didn't fight Ralph (Leer+Mud Slap spam) (teach Water Gun over Mud Slap)
+
+Hiker Russell:  Water Gun 3x
+
+ <details>
+ <summary>(optional)cave encounterless [f3-5 on Sandshrew Manip](https://pastebin.com/vuQixFYT) </summary>
+ <br>
+ <img src="https://cdn.discordapp.com/attachments/751229264816504932/757277654679289977/uce.png"/>
+ </details>
+
+Firebreather Ray: Water Gun, Scratch (Rage if Vulpix is in Redbar)
+
+If you get Burned by Ember, pick up and use Hidden Full Heal
+
+Azalea Town/Slowpoke Well:
+
+Avoid Spinner Anthony (incase you hit, spam Water Gun)
+
+Shop after talking to Kurt
+
+- 3 Super Potions (3D)
+- 3 Repels (1D)
+- 3 Antidotes (1D)
+- 1 Paralyz Heal (1D)
+
+go talk to Kurt
+Enter Slowpoke Well
+
+Rocket Grunt #1: spam Water Gun (Scratch instead if you are gonna hit L15 here)
+
+Heal to 15+ HP
+
+Rocket Grunt #2:
+- Zubat: Water Gun 2x
+  - Scratch if already L15
+- Ekans: Scratch 3x
+
+Heal to 10+ HP
+
+Rocket Grunt #3:
+- Rattata: Spam Rage
+  - if you don’t get a Rage stack on Rat Water Gun Zubats
+- Zubats: Spam Rage
+
+Heal to 11+ HP if level up before, 16+HP if didn’t
+
+Rocket Grunt #4: Water Gun 3x/4x
+
+Enter Bugsy's Gym (Kurt healed you), fight right twin
+
+Twins Amy & May:
+- Ledyba: Scratch 2x
+- Spinarak: Water Gun 2x
+
+Do not heal if poisoned, go left
+
+Bug Catcher Josh:
+- Leer + Scratch x2-3
+  - (use Paralyz Heal turn 3 or after fight)
+
+
+If below 30HP: Heal, (heal Poison) when passing Benny
+
+Avoid Spinner Benny (if hit: Water Gun, Water Gun + Rage, spam Rage)
+
+Gym Leader Bugsy: spam Rage
+
+Menu Before Rival 2:
+- Use a Super Potion / Potion if it heals full
+- Repel (1D)
+- (Heal Poison)
+- Teach Fury Cutter over Rage (Slot3)
+
+Rival 2:
+- Gastly: Water Gun x2
+- Bayleef: Furry Cutter x3 (+ Scratch)
+- Zubat: (Heal Poison if below half HP) Scratch x2-3 if no Reflect/Growl, otherwise Water Gun x2-3
+
+Ilex Forrest:
+
+Do Farfetch’d Quest
+Get Cut from the left guy, when repel runs out:
+- Repel (1D if Supers in Slot 1)
+- Teach Cut to cutter (L)
+
+Exit Ilex Forrest, repel one more time, get Kenya
+
+Enter Underground
+
+Fight Slowpokes: spam Fury Cutter
+
+pick up coin case from above (IMPORTANT)
+Equip Berry to Toto, Swap Toto with kenya
+walk left 1, up 1, left 2
+Open pokedex, scroll with right until bellsprout, open summary to hear its cry
+use coin case
+walk down, left, up
+talk to red (press A)
+
+
+Notes for coin case setup:
+- Croconaw always needs to be 4th in part and its movesets needs to be Scratch/Leer/FuryCutter/WaterGun, in this specific order
+- Either Egg or Kenya in 3rd is ~71% to work with random DVs (L6 Bellsprout is 71%*(12/16) to work, L3 Bellsprout never works
+- TODO: check sandshrew chances
+
+If you want to finish the run save after getting coin case, and if the glitch fails try different pokemons in slot3.
```

Comment:
suggesting: If you want to finish the run, save after getting coin case. If the glitch fails try different pokemons in slot3.